### PR TITLE
Add omnisharp language server (C# LSP)

### PR DESCRIPTION
**Pull request by D0nKarnag3**
Added C# LSP using omnisharp

### CODE REVIEW

**Review by erkrnt — APPROVED** [review]

File: servers/omnisharp/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM debian:bullseye-slim
+
+WORKDIR /tmp
+
+RUN apt update \
```
Comment (erkrnt) [inline servers/omnisharp/Dockerfile:5]: Update this line from `apt` to `apt-get` to fix linting errors: https://github.com/lspcontainers/dockerfiles/runs/3526706197#step:3:13

Comment (D0nKarnag3) [reply servers/omnisharp/Dockerfile]: Changed apt to apt-get as suggested.

File: servers/omnisharp/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM debian:bullseye-slim
+
+WORKDIR /tmp
+
+RUN apt-get update \
+  && apt install curl -y \
```
Comment (erkrnt) [inline servers/omnisharp/Dockerfile:6]: One more change please - gotta get all instances of it.

File: servers/omnisharp/Dockerfile (hunk 1 above)
Comment (erkrnt) [reply servers/omnisharp/Dockerfile]: Awesome - thank you! Got one more line right under - needs all instances updated.

File: servers/omnisharp/Dockerfile (hunk 2 above)
Comment (D0nKarnag3) [reply servers/omnisharp/Dockerfile]: Sorry - I should have noticed that.

**Comment by erkrnt** [comment]
This is perfect - I'm going to go ahead and merge this as I need to make some changes to the linting rules. Thank you for the updates!!

**Pull request merged**